curriculum/challenges/_meta/learn-localstorage-by-building-a-todo-app/meta.json

```
@@ -0,0 +1,249 @@
+{
+  "name": "Learn localStorage by Building a Todo App",
+  "isUpcomingChange": true,
+  "usesMultifileEditor": true,
+  "hasEditableBoundaries": true,
+  "dashedName": "learn-localstorage-by-building-a-todo-app",
+  "order": 42,
```

```suggestion
  "order": 41,
```
This is cause of no redirect to correct next challenge after finishing step. There's another project with order `42` what apparently creates conflict.

---

Thanks for catching that, Krzysztof.